backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/select/BpkSelect.kt

```diff
@@ -0,0 +1,44 @@
+/**
+ * Backpack for Android - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.skyscanner.backpack.compose.select
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import net.skyscanner.backpack.compose.select.internal.BpkSelectImpl
+
+sealed interface BpkSelectState {
```

Comment:
we can just reuse the state from `FieldSet` - I know we technically don't have the validated state here, but otherwise the API between fieldset/select will be misaligned and we won't be able to pass through the state between fieldset & select